front/playwright.config.ts

```diff
@@ -12,31 +12,33 @@ export default defineConfig({
   testDir: './tests',
 
   /* Maximum time one test can run for. */
-  timeout: process.env.CI ? 90 * 1000 : 180 * 1000, // 90 seconds in CI, otherwise 180 seconds
+  timeout: 90_000,
   expect: {
     toHaveScreenshot: { maxDiffPixelRatio: 0.02 },
     /**
      * Maximum time expect() should wait for the condition to be met.
      */
-    timeout: process.env.CI ? 10 * 1000 : 30 * 1000, // 10 seconds in CI, otherwise 30 seconds
+    timeout: 10_000,
   },
 
   /* Run tests in files in parallel */
   fullyParallel: true,
   /*
    * Limit parallelism in CI based on CPU capacity,
    * running 50% of the available workers when in CI.
-   * Otherwise, run tests with a single worker.
+   * Otherwise, run tests with 2 workers.
    */
-  workers: process.env.CI ? '50%' : 1,
+  workers: process.env.CI ? '50%' : 2,
```

Comment:
```suggestion
  workers:'50%',
```

I think the most important is for the CI, a developper can set a specific number of workers him/herself if needed